server/server_test.go

```diff
@@ -1180,17 +1179,6 @@ func createRandomOccurrence(kind grafeas_common_proto.NoteKind) *grafeas_proto.O
 	}
 }
 
-func structToJsonBody(i interface{}) io.ReadCloser {
-	b, err := json.Marshal(i)
-	Expect(err).ToNot(HaveOccurred())
-
-	return io.NopCloser(strings.NewReader(string(b)))
-}
-
-func createEsSearchResponse(occurrences []*grafeas_proto.Occurrence) io.ReadCloser {
-	return createPaginatedEsSearchResponse(occurrences, len(occurrences))
-}
-
```

Comment:
I removed some of these in #100 too, as well as `createPaginatedEsSearchResponse`.  we can merge this first and I can deal with those conflicts.

Reply:
I decided to just revert this, since it was in a standalone commit.